contracts/test/ERC20DestinationTests.t.sol

```diff
@@ -0,0 +1,124 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {IERC20BridgeTest} from "./IERC20BridgeTests.t.sol";
+import {TeleporterTokenDestinationTest} from "./TeleporterTokenDestinationTests.t.sol";
+import {ERC20Destination} from "../src/ERC20Destination.sol";
+import {IERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/utils/SafeERC20.sol";
+
+contract ERC20DestinationTest is IERC20BridgeTest, TeleporterTokenDestinationTest {
+    using SafeERC20 for IERC20;
+
+    ERC20Destination public app;
+
+    string public constant MOCK_TOKEN_NAME = "Test Token";
+    string public constant MOCK_TOKEN_SYMBOL = "TST";
+    uint8 public constant MOCK_TOKEN_DECIMALS = 18;
+
+    //solhint-disable func-named-parameters
```

Comment:
Rather than disabling this lint warning, can we just used named params? Even in unit tests, it can be helpful for readability and avoiding parameter ordering mishaps. https://docs.soliditylang.org/en/latest/control-structures.html#function-calls-with-named-parameters